src/components/button/component.js

```diff
@@ -4,16 +4,17 @@ import './styles.scss';
 
 export default (props) => {
   const { children,
-    isDisabled,
     className,
+    dark,
+    light,
     ...domProps } = props;
 
   return (
     <button
       type="button"
       className={classnames('c-button', {
-        ['disabled']: isDisabled,
-        [className]: className
+        'dark': dark,
```

Comment:
I would recommend you not doing this, go back to the previous state. The thing is buttons are not light or dark, they have also sizes, states... You can't create a button by defining 100000 props